.gitignore

```diff
@@ -44,8 +44,7 @@ object_storage/
 __emails__
 share/GeoLite2-City.mmdb
 a11y/
-# pyenv local config
-.python-version
 /test-results/
 /playwright-report/
 playwright/e2e-vrt/**/*-darwin.png
+gen
```

Comment:
If antlr fails, it generates some intermediary Java source that it dumps here 🤷 . It's easy to accidentally commit, so ignoring.

Reply:
Can you also add a comment here? This `gen` will be totally cryptic down the line – in a few years someone will have to scroll through this PR in hope of figuring out when this is ever needed.